docs/user/set-up/install-agent-macos.md

```diff
@@ -0,0 +1,40 @@
+---
+aliases:
+- /docs/agent/latest/set-up/install-agent-macos/
+title: Install Grafana Agent on macOS
+weight: 130
+---
+## Install Grafana Agent on macOS
+
+Install Grafana Agent and get it up and running on macOS. 
+
+### Overview
+Use Homebrew to install the most recent released version of Grafana using the Homebrew package. You can also install Grafana Agent on macOS using the macOS binary.
+
+### Steps
+
+1. Open a terminal and enter:
+   
+   `brew update
```

Comment:
For multiline code blocks to appear, you need to get them into triple-backtick fence blocks, or idented by four spaces.

````
```
this will become 
a multi-line code
block
```
````